sdk/maps/azure-maps-timezone/README.md

```diff
@@ -45,16 +45,35 @@ Then pass an `AZURE_SUBSCRIPTION_KEY` as the `credential` parameter into an inst
 import os
 
 from azure.core.credentials import AzureKeyCredential
-from azure.maps.timezone import MapsTimezoneClient
+from azure.maps.timezone import MapsTimeZoneClient
 
 credential = AzureKeyCredential(os.environ.get("AZURE_SUBSCRIPTION_KEY"))
 
-timezone_client = MapsTimezoneClient(
+timezone_client = MapsTimeZoneClient(
     credential=credential,
 )
 ```
 
-#### 2. Authenticate with an Microsoft Entra ID credential
+#### 2. Authenticate with a SAS Credential
+
+You can authenticate with your generated SAS token.
+Once the SAS token is created, set the value of the token as environment variable: `AZURE_SAS_TOKEN`.
+Then pass an `AZURE_SAS_TOKEN` as the `credential` parameter into an instance of AzureSasCredential.
```

Comment:
The description in this paragraph is incomplete. We need to tell user to install `azure-mgmt-maps`:
```bash
pip install azure-mgmt-maps
```

And tell them how to get the correct `AZURE_SAS_TOKEN` value using `list_sas` method from `azure-mgmt-maps`.

You can refer to [.NET SDK](https://github.com/Azure/azure-sdk-for-net/tree/main/sdk/maps/Azure.Maps.Search#shared-access-signature-sas-authentication) for a more complete SAS token example code snippet.